crypto/include/rdbx.h

```diff
@@ -109,7 +109,7 @@ srtp_err_status_t srtp_rdbx_dealloc(srtp_rdbx_t *rdbx);
  * index to which s corresponds, and returns the difference between
  * *guess and the locally stored synch info
  */
-int32_t srtp_rdbx_estimate_index(const srtp_rdbx_t *rdbx,
+ssize_t srtp_rdbx_estimate_index(const srtp_rdbx_t *rdbx,
```

Comment:
not sure this should be ssize_t ...

Reply:
Yeah, I'm not sure what we should use here. The int32_t type used before was fine. However, I saw a mix of types used (e.g., int used in rdbx_check_add() when receiving the result of the call to srtp_index_guess() that returned an int32_t, too).  This type indicates that it's signed and it arguably bigger than it needs to be.  Perhaps we should revisit this and create a "SeqDelta" type or something that is sized appropriately?  I'd prefer to not add more to this right now, but that might be useful.